cypress/integration/inventory/Item-status-date-updates.spec.js

```diff
@@ -0,0 +1,376 @@
+              .then(organization => {
+                order.vendor = organization.id;
+                orderLine.physical.materialSupplier = organization.id;
+                orderLine.eresource.accessProvider = organization.id;
+              });
+            cy.getMaterialTypes({ query: 'name="book"' })
+              .then(materialType => { orderLine.physical.materialType = materialType.id; });
+          });
+        effectiveLocationServicePoint = NewServicePoint.getDefaulServicePoint();
+        effectiveLocationServicePointName = effectiveLocationServicePoint.body.name;
+        const effectiveLocationServicePointId = effectiveLocationServicePoint.body.id;
+        notEffectiveLocationServicePoint = NewServicePoint.getDefaulServicePoint();
+        notEffectiveLocationServicePointName = notEffectiveLocationServicePoint.body.name;
+
+        Institutions.createViaApi(Institutions.defaultUiInstitutions.body);
+        const specialCampuse = { ...Campuses.defaultUiCampuses.body };
+        specialCampuse.institutionId = Institutions.defaultUiInstitutions.body.id;
+        Campuses.createViaApi(specialCampuse);
+        const specialLibrary = { ...Libraries.defaultUiLibraries.body };
+        Libraries.createViaApi(specialLibrary);
+        specialLibrary.campusId = specialCampuse.id;
+
+        effectiveLocation = { ...NewLocation.getDefaultUiLocation(Institutions.defaultUiInstitutions.body.id, specialCampuse.id, specialLibrary.id).body, servicePointIds: [effectiveLocationServicePoint.body.id], primaryServicePoint: effectiveLocationServicePoint.body.id };
+
+        userRequestPreferences = {
+          id: uuid(),
+          fulfillment: 'Delivery',
+          defaultDeliveryAddressTypeId: null,
+          defaultServicePointId: effectiveLocationServicePointId,
+          delivery: true,
+          holdShelf: true,
+          userId: userForDeliveryRequest.id,
+        };
+
+        ServicePoints.createViaApi(effectiveLocationServicePoint.body);
+        ServicePoints.createViaApi(notEffectiveLocationServicePoint.body);
+        cy.addServicePointToUser([effectiveLocationServicePoint.body.id, notEffectiveLocationServicePoint.body.id],
+          user.userId, effectiveLocationServicePoint.body.id);
+        Locations.createLocationViaApi(effectiveLocation)
+          .then(locations => {
+            orderLine.locations[0].locationId = locations.body.id;
+            itemLocation = locations.body.name;
+          });
+        cy.login(userProperties.username, userProperties.password)
+          .then(() => {
+            Orders.createOrderWithOrderLineViaApi(order, orderLine)
+              .then(orderNumber => {
+                createdOrderNumber = orderNumber;
+              });
+          });
+        cy.getUsers({ limit: 1, query: `"personal.lastName"="${user.username}" and "active"="true"` })
+          .then((users) => {
+            userBarcode = users[0].barcode;
+          });
+      });
+
+    cy.createTempUser([
```

Comment:
can you work through diku_admin instead of additional users?

Reply:
no, I add addresses to user for test